### ISSUE TITLE
Patch for /AndorsTrail/res/values-ru/content_conversationlist.xml

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#309]
```
orfo
```

Original issue reported on code.google.com by `konst...@gmail.com` on 30 Jun 2012 at 8:00

Attachments:
- [content_conversationlist.xml.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-309/comment-0/content_conversationlist.xml.patch)

**Comment by GoogleCodeExporter** [comment]
```
Merged in 3ac6a15e6c0f36fd51fe0c27bca7c36080bab62f. Thanks konstmih!
```

Original comment by `oskar.wi...@gmail.com` on 14 Sep 2012 at 6:55
- Changed state: **Ready**
- Added labels: **Type-Translation**, **Milestone-0.6.12**
- Removed labels: **Type-Patch**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.12.
```

Original comment by `oskar.wi...@gmail.com` on 28 Oct 2012 at 7:42
- Changed state: **Fixed**
